composer.json

```diff
@@ -20,18 +20,18 @@
     "require": {
         "php": "^8.0",
         "ext-mbstring": "*",
-        "psr/log": "^1.1|^2.0|^3.0",
-        "yiisoft/aliases": "^1.1|^2.0|^3.0",
+        "psr/log": "^3.0",
+        "yiisoft/aliases": "^3.0",
```

Comment:
Why drop support?

Reply:
Because they are versions, which have had bug fixes, and can cause a malfunction.

Reply:
This is a bad practice, having support for versions with bugs :)

Reply:
> 1. This is should be done in separate PR and have a line in changelog.
> 2. It's not `yiisoft/assets` package bugs. `yiisoft/assets` good work with droped versions.

The current version of asset with Aliases 2.0 won't work, because of the old configuration, so it's a bug :)

Reply:
1. No need for separate PR. Changelog entry is OK to add.
2. It makes sense to drop it because of config incompatibility.

Reply:
> It makes sense to drop it because of config incompatibility.

What configurations are we talking about?

- `psr/log` don't contain configuration;
- configuration of `yiisoft/aliases` does not intersect with `yiisoft/assets` configuration.

Reply:
Aliases:

3.0.

![image](https://github.com/yiisoft/assets/assets/42547589/2d2013c6-dced-435b-aada-aa0ba6e11d16)

2.0

![image](https://github.com/yiisoft/assets/assets/42547589/ff3da110-321e-40d4-a02c-53e504772df2)

Psr/Log:

yiisoft/log: (1.0.4)

![image](https://github.com/yiisoft/assets/assets/42547589/e6cbd67a-c0a5-4772-8b18-00e148681d07)

yiisoft/log: (2.0)

![image](https://github.com/yiisoft/assets/assets/42547589/f6cea6fa-c88b-4b0c-9adc-f8da288eb5c4)

Reply:
Anyway, I'd be happy to revert the composer.json changes, and leave the bug in the package, if you like.

Reply:
`yiisoft/log` it has nothing to do with `yiisoft/assets`.

Сonfiguration of `yiisoft/aliases` it has nothing to do with `yiisoft/assets`.

Part of `yiisoft/aliases` package that used in `yiisoft/assets` is not contain problems. All works is fine.

I see one reason for drop support of old version in this case: although `yiisoft/assets` works fine with older versions, but it package don't contain tests in CI for previous versions of `psr/log` and `yiisoft/aliases`. The rest are voiced arguments in this PR conversion is incorrect.

Reply:
> I see one reason for drop support of old version in this case: although `yiisoft/assets` works fine with older versions, but it package don't contain tests in CI for previous versions of `psr/log` and `yiisoft/aliases`. The rest are voiced arguments in this PR conversion is incorrect.

Thks, for correcting me.